### PR TITLE
fix: Disable manual typing values in dropdowns

### PR DESCRIPTION
**Pull request by GDamyanov**
Fix for:  #1544 

- Disable manual typing in dropdowns

### CODE REVIEW

**Review by nikmace — APPROVED** [review]
Small and simple change
Coverage not applicable
Tested manually ✅

File: .changeset/yellow-oranges-study.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ux/control-property-editor': patch
```
[the remark this answers is not in the document]
Comment (GDamyanov) [reply .changeset/yellow-oranges-study.md:2]: Done

**Review by nikmace — APPROVED** [review]
Small and simple change
Coverage not applicable
Tested manually ✅

File: .changeset/yellow-oranges-study.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (zdravko-georgiev) [reply .changeset/yellow-oranges-study.md:2]: thanks

**Review by zdravko-georgiev — APPROVED** [review]
Simple and straightforward change
Excellent test coverage
Manual testing ✅

**Review by voicis — APPROVED** [review]
Dropdowns are only used for enums, so we only need a fixed set of values.
- Simple change
- Tested locally
- Good test coverage

**Pull request merged**